spec/features/state_file/complete_intake_spec.rb

```diff
@@ -316,6 +316,88 @@
       expect(submission.submission_bundle).to be_present
       expect(submission.current_state).to eq("queued")
     end
+
+    it "properly calculates the sales/use tax", required_schema: "nc" do
+      visit "/"
+      click_on "Start Test NC"
+
+      expect(page).to have_text I18n.t("state_file.landing_page.edit.nc.title")
+      click_on I18n.t('general.get_started'), id: "firstCta"
+
+      step_through_eligibility_screener(us_state: "nc")
+
+      step_through_initial_authentication(contact_preference: :email)
+
+      check "Email"
+      check "Text message"
+      fill_in "Your phone number", with: "+12025551212"
+      click_on "Continue"
+
+      click_on I18n.t("general.accept")
+      click_on I18n.t("state_file.questions.terms_and_conditions.edit.accept")
+
+      step_through_df_data_transfer("Transfer Bert 1099 r")
+
+      select("Buncombe", from: "County")
+      click_on I18n.t("general.continue")
+
+      choose "state_file_nc_veteran_status_form_primary_veteran_no"
+      choose "state_file_nc_veteran_status_form_spouse_veteran_no"
+      click_on I18n.t("general.continue")
+
+
+      # Of the 4 buttons on this page, select the first button that says "Review and edit state info"
+      click_on I18n.t("state_file.questions.income_review.edit.review_and_edit_state_info"), match: :first
+      click_on I18n.t("general.continue")
+
+      click_on I18n.t("state_file.questions.income_review.edit.review_and_edit_state_info"), match: :first
+      expect(page).to have_text(strip_html_tags(I18n.t("state_file.questions.retirement_income.edit.title_html", payer_name: "Elmo Retirement Ltd")))
+      click_on I18n.t("general.continue")
+
+      click_on I18n.t("general.continue")
+      choose strip_html_tags(I18n.t("state_file.questions.nc_retirement_income_subtraction.edit.income_source_bailey_settlement_html"))
+      check "state_file_nc_retirement_income_subtraction_form_bailey_settlement_at_least_five_years"
+      check "state_file_nc_retirement_income_subtraction_form_bailey_settlement_from_retirement_plan"
+      click_on I18n.t("general.continue")
+      click_on I18n.t("general.continue")
```

Comment:
not sure where we are with the double continues, maybe a comment about which page were navigating thru here?